modules/system/version.nix

```diff
@@ -9,7 +9,7 @@ let
 
   parseGit = path:
     if pathExists "${path}/.git/" then rec {
```

Comment:
perhaps this one should also be removed then?

Reply:
Good point. I updated the PR.